webpack.config.babel.js

```diff
@@ -71,14 +71,16 @@ const PROD_CONFIG = {
   'JWT_FACTORY': 'https://token-factory.onfido.com/sdk_token',
   'DESKTOP_SYNC_URL' : 'https://sync.onfido.com',
   'MOBILE_URL' : 'https://id.onfido.com',
+  'SMS_DELIVERY_URL': 'https://telephony.onfido.com',
 }
 
 const STAGING_CONFIG = {
   'ONFIDO_API_URL': 'https://apidev.onfido.com',
   'ONFIDO_SDK_URL': 'https://sdk-staging.onfido.com',
   'JWT_FACTORY': 'https://token-factory-dev.onfido.com/sdk_token',
   'DESKTOP_SYNC_URL' : 'https://sync-dev.onfido.com',
-  'MOBILE_URL' : 'https://id-dev.onfido.com',
+  'MOBILE_URL' : 'http://192.168.64.2:8080',
```

Comment:
Not sure you meant to commit this...

Could you push your stuff to s3 staging too?

Reply:
No, I wasn't!